storage/src/vespa/storage/storageserver/servicelayernode.cpp

```diff
@@ -258,8 +262,9 @@ ServiceLayerNode::createChain()
         chain->push_back(StorageLink::UP(releaseStateManager().release()));
         return chain;
     }
+    MergeThrottler* merge_throttler;
     chain->push_back(StorageLink::UP(new OpsLogger(compReg, _configUri)));
-    chain->push_back(StorageLink::UP(new MergeThrottler(_configUri, compReg)));
+    chain->push_back(StorageLink::UP(merge_throttler = new MergeThrottler(_configUri, compReg)));
```

Comment:
I'm not a big fan of embedded assignments.

Reply:
Will pull assignment out of expression